### PR TITLE
Add `theme` prop to Icons

### PR DESCRIPTION
**Pull request by oliverabrahams**
## What are you changing?

- Added new theming to Icons

## Why?

-

### CODE REVIEW

**Review by mxdvl — COMMENTED** [review]
Am I correct in thinking that only monochrome icons can be themed?

**Comment by oliverabrahams** [comment]
> Am I correct in thinking that only monochrome icons can be themed?

Yes, I think that makes sense. What do you think?

**Comment by mxdvl** [comment]
> > Am I correct in thinking that only monochrome icons can be themed?
> 
> Yes, I think that makes sense. What do you think?

I think that enabling polychromatic icons to be styled is going to be a much larger endeavour that is best tackled only if consumers demand it.

File: libs/@guardian/source-react-components/src/icons/Icons.stories.tsx [hunk 1]
```diff
@@ -327,19 +331,6 @@ MediumIconsBrandTheme.parameters = {
 		default: 'brandBackground.primary',
 	},
 };
-MediumIconsBrandTheme.decorators = [
```
Comment (jamesmockett) [inline libs/@guardian/source-react-components/src/icons/Icons.stories.tsx:330]: Glad to see these style overrides removed and replaced with a supported method so we're not encouraging people to do the wrong thing 🙈

**Review by jamesmockett — APPROVED** [review]
Nice work 🚀

**Pull request merged**